client/components/home.jsx

```diff
@@ -1,70 +1,45 @@
 import React, {PropTypes} from "react";
 import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
-import {Card, CardHeader, CardText} from "material-ui/Card";
+import darkBaseTheme from "material-ui/styles/baseThemes/darkBaseTheme";
+import getMuiTheme from "material-ui/styles/getMuiTheme";
```

Comment:
@ananavati, is it convention to capitalize *all* imports? darkBaseTheme and getMuiTheme are not react components.